server/app.js

```diff
@@ -45,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/notification", notificationRouter);
```

Comment:
From the aformentioned [article](https://restfulapi.net/resource-naming/): 
> Use the “plural” name to denote the collection resource archetype.

```suggestion
app.use("/notifications", notificationRouter);
```